packages/core/src/internal-components/popup/pointer.element.ts

```diff
@@ -18,13 +18,12 @@ import styles from './pointer.element.scss';
  * <cds-internal-pointer></cds-internal-pointer>
  * ```
  *
- * @beta
+ * @internal
```

Comment:
Pointer is part of the dropdown work. It should still be beta as well.

Reply:
reverted